pkgs/development/libraries/qt-6/default.nix

```diff
@@ -90,7 +98,9 @@ let
       qt3d = callPackage ./modules/qt3d.nix { };
       qt5compat = callPackage ./modules/qt5compat.nix { };
       qtcharts = callPackage ./modules/qtcharts.nix { };
-      qtconnectivity = callPackage ./modules/qtconnectivity.nix { };
+      qtconnectivity = callPackage ./modules/qtconnectivity.nix {
```

Comment:
You must use the callPackage wrapper or integrate it into qt's callPackage wrapper.

Reply:
It actually uses the same callPackage wrapper as in qtbase.